src/vue/common/Pagination.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
I vote for calling the component `collection-renderer` or smth like that.

Reply:
Fixed